src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -382,52 +382,52 @@
   product(bool, UseAdaptiveSizePolicyFootprintGoal, true,                   \
           "Use adaptive minimum footprint as a goal")                       \
                                                                             \
-  product(uintx, AdaptiveSizePolicyWeight, 10,                              \
+  product(uint, AdaptiveSizePolicyWeight, 10,                               \
           "Weight given to exponential resizing, between 0 and 100")        \
           range(0, 100)                                                     \
                                                                             \
-  product(uintx, AdaptiveTimeWeight,       25,                              \
+  product(uint, AdaptiveTimeWeight,       25,                               \
           "Weight given to time in adaptive policy, between 0 and 100")     \
           range(0, 100)                                                     \
                                                                             \
-  product(uintx, PausePadding, 1,                                           \
+  product(uint, PausePadding, 1,                                            \
           "How much buffer to keep for pause time")                         \
-          range(0, max_juint)                                               \
+          range(0, UINT_MAX)                                                \
```

Comment:
Why did you change from max_juint -> UINT_MAX ?  max_juint should have compile without integer size mismatch.

Reply:
Or, not in this change, but should we prefer UINT_MAX (found in /usr/include/limits.h) to our globalDefinitions.hpp definitions?

Reply:
Now I see Thomas's comments about this.  I guess changing a couple at a time would be ok.